plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -95,6 +96,18 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Accumulo connector does not support column default values");
     }
 
+    @Override
+    public void testCreateTableWithColumnComment()
+    {
+        // TODO Avoid setting hard-coded column comment
+        String tableName = "test_create_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint COMMENT 'test comment')");
+        assertEquals(getColumnComment(tableName, "a"), "Accumulo row ID");
```

Comment:
What is happening here? Why is the comment of the table "Accumulo row ID" ?

Reply:
The connector ignores comments in DDL and sets prepared comment in https://github.com/trinodb/trino/blob/master/plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/AccumuloClient.java#L324-L354